projects/plugins/jetpack/modules/seo-tools/class-jetpack-seo-posts.php

```diff
@@ -80,6 +106,17 @@ public static function register_post_meta() {
 			),
 		);
 
+		$html_title_args = array(
+			'type'         => 'string',
+			'description'  => __( 'Custom title to be used in HTML <title /> and <meta /> tag.', 'jetpack' ),
```

Comment:
This string is incorrect - the custom title will only be used in the `<title>` tag.